account_invoice_line_sequence/models/account_move.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    sequence_show = fields.Integer(
+        help="Shows the sequence of this line in the " " invoice.",
```

Comment:
```suggestion
        help="Shows the sequence of this line in the invoice.",
```
please, do this in all the cases where it happens